cmd/runner/executetest.go

```diff
@@ -49,14 +50,15 @@ func ExecuteTest() *cobra.Command {
 				os.Exit(exitCodeConfigError)
 			}
 
-			if !executeTest(cmd.Context(), k, plan) {
+			if !executeTest(cmd.Context(), k, plan, probeImage) {
 				os.Exit(exitCodeFailure)
 			}
 		},
 	}
 
 	cmd.Flags().StringVarP(&testFile, "file", "f", "", "Path to the test configuration YAML file")
 	cmd.MarkFlagRequired("file") //nolint:errcheck
+	cmd.Flags().StringVar(&probeImage, "probe-image", "", fmt.Sprintf("Default probe image to use if test plan doesn't specify one (defaults to grafana/nethax-probe:%s')", kubernetes.ProbeImageVersion))
```

Comment:
🤔 how about if instead of showing the default in the flag description, we actually have a default? Also, perhaps we need to rename this flag to make it more descriptive:

```suggestion
	cmd.Flags().StringVar(&probeImage, "default-probe-image", "grafana/nethax-probe:"+kubernetes.ProbeImageVersion, "Default probe image to use if test plan doesn't specify one")
```

Reply:
lol this is so obvious and good I cannot believe I didn't think of this 🤦🏼‍♀️ 

Thank you!!!